irohad/consensus/yac/yac_hash_provider.hpp

```diff
@@ -52,6 +42,9 @@ namespace iroha {
          */
         std::string block_hash;
 
+        /// Consensus round
+        ordering::transport::Round round;
```

Comment:
No need to add this field in this pull request if storage and transport logic is unchanged.